### ISSUE TITLE
[Bug] Connect - Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by rasta-mouse** [cobbr/Covenant#75]
**Feature Request or Bug**
Bug

**Describe the feature request or bug**
The `Connect` Task from the UI returns:
```
Task Exception: Object reference not set to an instance of an object.
   at GruntExecutor.Grunt.TaskExecute(TaskingMessenger messenger, GruntTaskingMessage message)
```

**To Reproduce**
1. Execute GruntSMB stager on target host
2. HTTP Grunt > Task > Connect
3. Enter ComputerName & PipeName
4. Click Task

The connect command on the CLI works as expected.

**Expected behavior**
HTTP Grunt should connect and to the SMB Grunt

**Screenshots**
![image](https://user-images.githubusercontent.com/7346521/65608147-a7d71780-dfa5-11e9-92b0-02fb44a27b3f.png)

![image](https://user-images.githubusercontent.com/7346521/65608209-c63d1300-dfa5-11e9-8282-fb2cfd8d70a1.png)

**Covenant Server Information:**
 - OS: Kali Linux Rolling 2019.4
 - Docker or Native: Native

**Browser Information:**
 - Browser: Chrome
 - Version: 77.0.3865.90 (Official Build) (64-bit)

**Target Information (System that implant is running on):**
 - OS: Windows 10
 - Version: 1809 (Build 17763.737)

**Additional context**
HTTP and SMB Grunt are running on the same machine (e.g executing an SMB Grunt via a Priv Esc exploit).

**Comment by cobbr** [comment]
Confirmed. This happens *only* when using the Task tab, works fine from Interact tab.

**Comment by cobbr** [comment]
Thanks for the heads up @rasta-mouse! This ended up being a more pervasive issue than I thought. A few of the tasks that are handled differently than standard tasks were not working in the Task tab.

Should be fixed in 0123475bebd7f2ea8c56c0698a8a335884c7ca1c

**Comment by rasta-mouse** [comment]
Looks like it was a bit of pain.  Nice fix 👍
